### ISSUE TITLE
Accessing phpmyadmin

### ISSUE THREAD

**Issue by prabhuchelladurai** [r-baker/simple-magento-vagrant#33]
How could one access the phpmyadmin? 

I tried with 127.0.0.1:8080/phpmyadmin and got 404 Not Found error.

What is the address to be applied?

Thanks.

**Comment by rjbaker** [comment]
The provisioning script doesn't install PHPMyAdmin. You'll need to install it on the vagrant box first. Then it should be available using whatever path you configure.

**Comment by prabhuchelladurai** [comment]
Could you please guide me in installing PHPMyAdmin with step by step commands? 

Thanks.

**Comment by rjbaker** [comment]
Here's a guide: http://www.liquidweb.com/kb/how-to-install-and-configure-phpmyadmin-on-ubuntu-14-04/

Before you do this, make sure you're logged into the vagrant instance by typing `vagrant ssh`

**Comment by prabhuchelladurai** [comment]
Thankyou very much. I'll try this and get back to disturb you, if I happen to face a closed end... :)

**Comment by prabhuchelladurai** [comment]
I tried that and facing problems which I could not resolve. Database administrative user password is not accepted and the access is denied. I gave root password and some other options like, admin, root, password123123, password but nothing worked. I finally chose to select the option ignore. So, the installation was done flushing administrative password. Now, I tried to edit apache2.conf in vain as I am not able to edit the file even with sudo option.

Your help is much sought in this regard.

Thanks a lot.
